content/tutorials/manage-the-rippled-server/troubleshooting/understanding-log-messages.md

```diff
@@ -154,6 +154,10 @@ This is not strictly a problem, but if you want to acquire ledger history faster
 s2.ripple.com 51235
 ```
 
+## Potential Censorship
+
+The transaction censorship detector issues log messages when it detects potential transaction censorship. For more information about the detector and these log messages, see [Transaction Censorship Detection](transaction-censorship-detection.html).
+
```

Comment:
Please include an example of the log messages in question here, as demonstrated in the other examples on the page. The intent is for this page to be searchable by Ctrl-F if you have any log messages you don't understand.